chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/surveillancereportingactivity/excel/CellHelper.java

```diff
@@ -0,0 +1,55 @@
+package gov.healthit.chpl.scheduler.job.surveillancereportingactivity.excel;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.Date;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellStyle;
+
+public class CellHelper {
```

Comment:
Do we care about these checkstyle issues? I think there was one here as well as one in one of the XML files